improver_tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -443,6 +473,19 @@ def test_weights_sum_to_1(self):
         plugin = WeightedBlendAcrossWholeDimension(coord)
         plugin.check_weights(self.weights3d.data, 0)
 
+    def test_weights_sum_to_1_but_with_a_zero_weight(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5820979)